packages/pigeon/lib/dart_generator.dart

```diff
@@ -173,7 +173,15 @@ final BinaryMessenger? _binaryMessenger;
       if (func.arguments.isNotEmpty) {
         String argNameFunc(int index, NamedType type) =>
             _getSafeArgumentName(index, type);
-        final Iterable<String> argNames = indexMap(func.arguments, argNameFunc);
+        final Iterable<String> argNames =
```

Comment:
Should this be called `argValues` or `argValueExpressions` or something like that? `foo.index` isn't the name of an argument.

Reply:
Done, `argExpressions`